### PR TITLE
Document the 'pool/set-rf' command

### PR DESCRIPTION
**Pull request by lmlg**

### CODE REVIEW

File: docs/reference/commands/pool.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+.. code-block:: none
+
+   microceph disk [command]
+
+Available commands:
+
+.. code-block:: none
+
+   set-tf      Set the replication factor for pools
```
Comment (MggMuggins) [inline docs/reference/commands/pool.rst:17]: Typo?

**Review by MggMuggins — CHANGES_REQUESTED** [review]

**Review by UtkarshBhatthere — APPROVED** [review]

File: docs/reference/commands/pool.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+``pool``
+========
+
+Manages pools in MicroCeph.
+
+Usage:
+
+.. code-block:: none
+
+   microceph disk [command]
```
Comment (UtkarshBhatthere) [inline docs/reference/commands/pool.rst:11]: incorrect command.

**Review by UtkarshBhatthere — CHANGES_REQUESTED** [review]
one change needed.

**Review by UtkarshBhatthere — APPROVED** [review]
LGTM

**Pull request merged**